pkg/controllers/reconciler.go

```diff
@@ -159,6 +166,35 @@ func (r *reconciler) createObjects(ctx context.Context,
 	return result, nil
 }
 
+func UpgradeImages(image *string, initimage *string) (upgrade bool) {
+	for _, s := range []*string{image, initimage} {
+		if s == nil {
+			continue
+		}
+
+		if parts := strings.SplitN(filepath.Base(*s), ":", 2); len(parts) == 2 && len(parts[0]) > 0 {
+			name, version := parts[0], parts[1]
+			if ver, err := versionutil.ParseSemantic(version); err == nil && ver.LessThan(ImageMinVersion) {
+				*s = "intel/" + name + ":" + ImageMinVersion.String()
```

Comment:
we cannot change the repo org field. `intel/` is the default but users may have their own.